docs/section23/Rule23-12.md

```diff
@@ -0,0 +1,33 @@
+
+# Airside System - Rule 23-12 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** FALSE  
+**Rule ID:** 23-12  
+**Rule Description:** System 11 in heating mode SAT shall be modulated to maintain space temp and airflow shall be fixed at minimum airflow.    
```

Comment:
I am not sure if this matters but in case it does suggest not using abbreviations in the rule description (SAT and temp).

Reply:
sounds good.  Changed